lnbits/extensions/lnaddress/views_api.py

```diff
@@ -150,7 +151,7 @@ async def api_lnaddress_make_address(
         )
 
     domain_cost = domain.cost
-    sats = data.sats
+    sats = max(data.sats, 1) #Min amount is 1sat because 0sats invoice are not payable
```

Comment:
Again, third time this appears.

My feeling is that the minimum 1 sat rule should be enforced in the frontend and there should be a check here that throws an error if it is not. It's not good practice to overwrite variables (adds confusion, hard to debug).

Reply:
That's what I initially did (enforcing 1 sat on the frontend side) but this line: https://github.com/lnbits/lnbits/blob/5dda7eb2b9ad57b850a042aaeaf8d6165ac3ee2c/lnbits/extensions/lnaddress/views_api.py#L157 prevents it. I'm not really sure why it's there and did not want to risk refactoring it.

Reply:
I would guess that this is a sanity check. Here, you could add the `data.sats == 0 or` check for example.